espnet2/bin/hugging_face_export_vocabulary.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
```

Comment:
Does this go through some tests?

Reply:
Yes, this test file was added:
https://github.com/espnet/espnet/pull/4099/files/a1c837ae3bef371c564739f22a47c90610b63bff#diff-b42857d4cdb088a810a95a6784439e28acfa9d742d9f632e4c1b823566ec27b5